assume/world.py

```diff
@@ -36,7 +37,6 @@
 
 file_handler = logging.FileHandler(filename="assume.log", mode="w+")
 stdout_handler = logging.StreamHandler(stream=sys.stdout)
-stdout_handler.setLevel(logging.WARNING)
```

Comment:
Before, we always only warnings in stdout as well.
And LogLevel did only set the output level in `assume.log`

We can change this to show INFO if you want to :)